docs/build/guides/tokens/custom-sac-admin.mdx

```diff
@@ -0,0 +1,161 @@
+
+First, we'll enable the SAC contract using the Stellar CLI:
+
+```shell
+stellar contract asset deploy \
+    --source starIssuer --network testnet \
+    --asset STAR:GCS5NEHKJALCSVJAKIORXXVS554QQV5FNDLBK33CCAH6UIRYPXYZFC34
+# CBVYF2KJ72BRPLVPCUL3PGWDO5RK2XP4AJDHKX7GDDBJW42L2C6VT3SF
+```
+
+This gives us the SAC address of `CBVYF2KJ72BRPLVPCUL3PGWDO5RK2XP4AJDHKX7GDDBJW42L2C6VT3SF`.
+
+:::note
+
+We're using the asset's issuer account here to enable the SAC on the Testnet network, but this could be done using _any_ account.
+
+:::
+
+### Check the initial `Admin` address
+
+Now that we have the SAC deployed to the network, we can invoke the `admin` function to find out what its current `Admin` address is:
+
+```shell
+stellar contract invoke \
+    --source starIssuer --network testnet \
+    --id CBVYF2KJ72BRPLVPCUL3PGWDO5RK2XP4AJDHKX7GDDBJW42L2C6VT3SF \
+    -- \
+    admin
+# GCS5NEHKJALCSVJAKIORXXVS554QQV5FNDLBK33CCAH6UIRYPXYZFC34
+```
+
+Unsurprisingly, it's set to the issuer's `GCS5...` account. Let's change that.
+
+### Set the `Admin` address to a new `G...` account
+
+We can set this `Admin` address to any account that we want. Including an account that's un-funded on the network. As long as you're able to provide a valid signature for the account, which would take into consideration any multisig or signature weights that have been set up, you can use this event for future administration of the asset from within the contract.
+
+:::info
+
+Ensure that whatever account you're using for this `Admin` address has a way to sign transactions. That would mean you have a secret key or seed phrase for it.
+
+:::
+
+```sh
+stellar contract invoke \
+    --source starIssuer --network testnet \
+    --id CBVYF2KJ72BRPLVPCUL3PGWDO5RK2XP4AJDHKX7GDDBJW42L2C6VT3SF \
+    -- \
+    set_admin --new_admin GBL3IQ6KA3I7CS7C5ABTCU7CJJA7X76SBZKTTQVJ4APW5646DXEIQKHB
+```
+
+Now, invoking the `admin` function of the contract will return this new `GBL3...` address, rather than the original issuer of the asset.
+
+```shell
+stellar contract invoke \
+    --source starIssuer --network testnet \
+    --id CBVYF2KJ72BRPLVPCUL3PGWDO5RK2XP4AJDHKX7GDDBJW42L2C6VT3SF \
+    -- \
+    admin
+# GBL3IQ6KA3I7CS7C5ABTCU7CJJA7X76SBZKTTQVJ4APW5646DXEIQKHB
+```
+
+:::info
+
+If you examine the asset's `GCS5...` issuer account, you won't see anything different. The signers/weights will be exactly the same. The only place this change is seen is _within_ the SAC for this asset.
+
+:::
+
+### Set the `Admin` address to some contract `C...` address
+
+For this part of the example, we'll use a very simple "minter" contract as the `Admin` address. It has a very simple implementation, and only one main function, `start_mint`:
+
+```rust
+#[contractimpl]
+impl Contract {
+    pub fn __constructor(env: Env, sac_address: Address) {
+        env.storage().instance().set(&symbol_short!("SAC_ADDR"), &sac_address);
+    }
+
+    pub fn star_mint(env: Env, to: Address, amount: i128) {
```

Comment:
Even though this is just a simple example,  I still don't think it's ever a good idea to post examples of permissionless operations. We should at least leave a disclaimer about that.

Reply:
good call. i'll try to improve this part of the document, and will probably ask you for some help and feedback about what i put together